.github/workflows/dependencies.yaml

```
@@ -0,0 +1,38 @@
+name: Update dependencies
+on:
+  schedule:
+    - cron: '0 3 * * 6'  # Run every saturday at 3 am.
+jobs:
+  update_dependencies:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.7']
```

I'd use our oldest supported version.
```suggestion
        python-version: ['3.6']
```